app/services/idv/steps/recover_verify_wait_step_show.rb

```diff
@@ -1,122 +0,0 @@
-module Idv
-  module Steps
-    class RecoverVerifyWaitStepShow < VerifyBaseStep
-      STEP_INDICATOR_STEP = :verify_info
-
-      def call
-        poll_with_meta_refresh(IdentityConfig.store.poll_rate_for_verify_in_seconds)
-
-        process_async_state(async_state)
-      end
-
-      private
-
-      def process_async_state(current_async_state)
-        if current_async_state.none?
-          mark_step_incomplete(:verify)
-        elsif current_async_state.in_progress?
-          nil
-        elsif current_async_state.timed_out?
-          flash[:error] = I18n.t('idv.failure.timeout')
-          delete_async
-          @flow.analytics.track_event(Analytics::PROOFING_RESOLUTION_TIMEOUT)
-          mark_step_incomplete(:verify)
-        elsif current_async_state.done?
-          async_state_done(current_async_state)
-        end
-      end
-
-      def async_state_done(current_async_state)
-        add_proofing_costs(current_async_state.result)
-        response = idv_result_to_form_response(current_async_state.result)
-        response = check_ssn(flow_session[:pii_from_doc]) if response.success?
-        summarize_result_and_throttle_failures(response)
-        delete_async
-
-        if response.success?
-          mark_step_complete(:verify_wait)
-        else
-          mark_step_incomplete(:verify)
-        end
-
-        response
-      end
-
-      def async_state
-        dcs_uuid = flow_session[recover_verify_document_capture_session_uuid_key]
-        dcs = DocumentCaptureSession.find_by(uuid: dcs_uuid)
-        return ProofingSessionAsyncResult.none if dcs_uuid.nil?
-        return ProofingSessionAsyncResult.timed_out if dcs.nil?
-
-        proofing_job_result = dcs.load_proofing_result
-        return ProofingSessionAsyncResult.timed_out if proofing_job_result.nil?
-
-        proofing_job_result
-      end
-
-      def delete_async
-        flow_session.delete(recover_verify_document_capture_session_uuid_key)
-      end
-
-      def idv_failure(result)
-        attempter_increment if result.extra.dig(:proofing_results, :exception).blank?
-        if attempter_throttled?
-          @flow.analytics.track_event(
-            Analytics::THROTTLER_RATE_LIMIT_TRIGGERED,
-            throttle_type: :idv_resolution,
-          )
-          redirect_to idv_session_errors_recovery_failure_url
-        elsif result.extra.dig(:proofing_results, :exception).present?
-          redirect_to idv_session_errors_recovery_exception_url
-        else
-          redirect_to idv_session_errors_recovery_warning_url
```

Comment:
I think the associated routes could be removed as well:

https://github.com/18F/identity-idp/blob/8815b8fb7bd2b0771831a702b954e668466fba1c/config/routes.rb#L289-L292

And one more related controller detail:

https://github.com/18F/identity-idp/blob/073bf899d3ea723346a0ffaba5c4fd42687e6f14/app/controllers/idv/session_errors_controller.rb#L12-L14

There's also a couple related routes in the Rack initializer that are already partly invalid:

https://github.com/18F/identity-idp/blob/073bf899d3ea723346a0ffaba5c4fd42687e6f14/config/initializers/rack_timeout.rb#L11-L13